input/resources/au-medicarecardnumber.xml

```diff
@@ -44,6 +44,14 @@
         <valueString value="32788511952" />
       </example>
       <maxLength value="11" />
+      <condition value="inv-mcn-01"/>
+        <constraint>
+        <key value="inv-mcn-01"/>
+        <severity value="error"/>
+        <human value="Medicare Card Number shall be at least 10 digits"/>
+        <expression value="$this.toString().length() >= 10"/>
```

Comment:
toString() is not necessary as Identifier.value is already a string.

Reply:
Also, think the extension http://hl7.org/fhir/StructureDefinition/minLength is supported and could be considered. Could you please reconsider available options and provide justification for the chosen approach?

Reply:
Thanks @dbojicic. Retested minLength and confirm that it is supported. Changed to use minLength extension - removes need for invariant, clearly renders in table view stating minimum length alongside max length, assures at least 10 characters as per resolution.

Reply:
Confirmed.